src/settings/_spacing.scss

```diff
@@ -17,6 +17,7 @@ $p-xl: $p-base * 3 !default;    // 24px
 $p-l: $p-base * 2 !default;     // 16px
 $p-m: $p-base !default;         // 8px
 $p-s: $p-base / 2 !default;     // 4px
+$p-xs: $p-base / 4 !default;    // 2px
```

Comment:
there is no xs https://github.com/SUI-Components/UX-Definitions/blob/master/Tokens/Margins-and-Paddings.md

Reply:
I know, but in MA they need it. Maybe we can add it because should be safe.

Reply:
Do we really need this value for a component? I will check with Diego